styles.css

```
@@ -42,21 +45,43 @@ ul {
   margin-bottom: 20px;
+  color: var(--text-color);
 }
 
 h1 {
-  font-size: 3rem;
+  font-size: 3.5rem;
+  background: linear-gradient(135deg, var(--primary-color), var(--secondary-color));
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-clip: text;
+  text-fill-color: transparent;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)